app/legacy_lib/query_donations.rb

```diff
@@ -5,11 +5,15 @@
 
 module QueryDonations
   # Export all donation data for a given campaign
+
   def self.campaign_export(campaign_id)
-    Psql.execute_vectors(
+    nonprofit_id = Campaign.find(campaign_id).nonprofit_id
+    currency = Nonprofit.find(nonprofit_id).currency_symbol
```

Comment:
I think this would be better as follows:
```suggestion
    currency = Campaign.find(campaign_id).nonprofit.currency_symbol
```